### ISSUE TITLE
Need help with loading 2nd bot

### ISSUE THREAD

**Issue by EbKMoBBB** [kolton/d2bot-with-kolbot#2773]
The 1st bot loads in just fine, but the 2nd will get a battle.net connection error. I have it set to run as admin, these are fresh keys from blizz. just wondering if i messed the setup part up. i also have a ton of other questions but i'd like to get this handled first.

**Comment by DarkHorseDre** [comment]
each bot needs its own account

**Comment by EbKMoBBB** [comment]
Yeah I had separate accounts. One wouldn't work so I just cloned the acct
that worked. Got the bots working, just gotta figure out the char config
